src/utils/semantics.utils.ts

```diff
@@ -72,7 +72,19 @@ export const itemDialog: Array<H5PField> = [
         field: {
           label: "Link",
           name: "link",
-          type: H5PFieldType.Text,
+          type: H5PFieldType.Group,
+          fields: [
+            {
+              label: "Label",
+              name: "label",
+              type: H5PFieldType.Text,
+            },
```

Comment:
We could use https://github.com/ndla-h5p/h5p-editor-uuid to add uuids to links. If so, we also need to add it to `editorDependencies` in library.json.

```suggestion
            {
              label: "Id",
              name: "id",
              type: H5PFieldType.Text,
              widget: "uuid",
            },
            {
              label: "Label",
              name: "label",
              type: H5PFieldType.Text,
            },
```

Reply:
That is, if we even want ids. Using the link as a key might be enough.